ortho_seq_code/orthogonal_polynomial.py

```diff
@@ -2,47 +2,80 @@
 from numpy.linalg import *
 import time
 import os
+import pandas as pd
 import ortho_seq_code.sr as sr
 from ortho_seq_code.constants_orthoseqs import *
+from ortho_seq_code.utils import get_seq_info
 import click
 import itertools
+from matplotlib import pyplot as plt
 
 
 def create_dir_if_not_exists(out_dir):
-    if not os.path.exists(out_dir):
-        os.makedirs(out_dir)
-    else:
-        print("Path {} already exists, might be overwriting data".format(out_dir))
+    if os.path.exists(out_dir):
```

Comment:
Can you please revert this function back to what it was?

I do not see the point of changing the function unless you are still debugging somehow which does not make sense to me i.e why would you compare out_dir to a Boolean string False?

Reply:
That was an old idea to address the bug, didn't work, I just deleted the string False in the most recent commit :)